app/models/message.rb

```diff
@@ -0,0 +1,11 @@
+class Message < ApplicationRecord
+  def self.random
+    Message.order('RANDOM()').first
+  end
+
+  def as_json(_options = {})
+    {
+      message: message
+    }
+  end
+end
```

Comment:
# Back-End
As far as your API endpoints, everything works perfectly. Well done 👏🏾 

- Messages from the database are displayed using `/api/v1/messages` endpoint 💯 

![Screenshot from 2022-11-12 11-25-06](https://user-images.githubusercontent.com/38088818/201468087-3bcfdca2-f3b7-4415-b812-2f8ce1a94d60.png)

[OPTIONAL] Please add a few messages on the `/db/seeds.rb` file that can be used to populate the database with random messages that will be picked up by the front end.

You can have something like this:

![Screenshot from 2022-11-12 11-35-45](https://user-images.githubusercontent.com/38088818/201468201-80035f01-392a-4a55-aa2a-b9a44823f483.png)

# Front-End

[REQUIRED] As for your front-end React application. There's nothing there but boilerplate code. Kindly create an application that will get the messages from the API, store them in the Redux store, and display a random message using the `Greeting` component.

I left a detailed comment on your [frontend Pull Request ](https://github.com/c3c1l1a/hello-react-front-end/pull/2#pullrequestreview-1178077691)for reference.

#### Snapshot of the front-end pull request comment

![image](https://user-images.githubusercontent.com/38088818/201468388-197ed627-1573-4145-8adb-dda2c757f590.png)